src/categories/clothing.md

```diff
@@ -0,0 +1,17 @@
+---
+title: 'Clothing'
+image: ../images/clothing.jpg
+intro: The best brands are ones that use ethical production and sustainable materials
+---
+
+Sustainable clothing comes in many shapes and forms. One example is through ethical manufacturing which respects both the environment as well as humans living in surrounding areas - such as avoiding the use of harmful dyes. Another form can come from clothing made with natural materials such as:
+ * Modal - a breathable material made from beech tree pulp
+ * Rayon/Viscose - a manufactured fibre made with wood and plant-based products
+ * Bamboo - the bamboo plant makes light-weight, anti-bacterial clothes
+ * Hemp - material made from a strain of a cannabis plant 
+ * Cotton - a traditional material that is grown from plants
```

Comment:
Cotton is pretty terrible https://www.worldwildlife.org/industries/cotton 

Something starting off as a plant doesnt make it environmentally friendly. :(